src/apps/virtio_net/virtio_net.lua

```diff
@@ -0,0 +1,76 @@
+-- Application to connect to a virtio-net driver implementation
+--
+-- Licensed under the Apache 2.0 license
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Copyright (c) 2015 Virtual Open Systems
```

Comment:
Just a small thing for us to think about in the future:

I would like to avoid these license and copyright lines in the source files. To me it is boilerplate that doesn't add value. We already spell out the license in the `COPYING` file and Git already tracks the authorship very directly. If those mechanisms are not sufficient for some reason then it would be best to open an Issue to discuss that rather than starting to add random notes in the source code.

Could be that we should make some improvements e.g. write a `CONTRIBUTING.md` that spells things out in more detail, perhaps add a file where people can note that they have assigned copyright for certain of their contributions to somebody else (e.g. employer), perhaps help contributors get more visibility for their contributions in some way, etc.

However for now there are few enough instances that it seems safe to ignore :). There are also important exceptions e.g. where we have imported some code that is MIT licensed that may have implications one day.